src/devices/LidarLiteV3/LidarLiteV3.cs

```diff
@@ -0,0 +1,358 @@
+        /// repetition finishes (forever if infinite).
+        /// </remarks>
+        /// <param name="withReceiverBiasCorrection">Faster without bias correction, but more prone to errors if condition changes.</param>
+        /// <returns>Distance in cm</returns>
+        public int MeasureDistance(bool withReceiverBiasCorrection = true)
+        {
+            return MeasureDistanceAsync(withReceiverBiasCorrection).Result;
+        }
+
+        /// <summary>
+        /// Set the repetition mode to enable automatic measurement.
+        /// </summary>
+        /// <param name="measurementRepetition">Repetition mode, either Off, Repeat, or RepeatInfinitely.</param>
+        /// <param name="count">If Repeat, the number of times to repeat the measurement.</param>
+        /// <param name="delay">The delay between each measurements. Note the unit does not directly to hz, a value of 20 maps to about 100 hz.</param>
+        public void SetMeasurementRepetitionMode(MeasurementRepetition measurementRepetition, int count = -1, int delay = -1)
+        {
+            if (count < 2 || count > 254)
+            {
+                throw new ArgumentOutOfRangeException("Count must be between 2 and 254");
+            }
+
+            switch (measurementRepetition)
+            {
+                case MeasurementRepetition.Repeat:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, (byte)count);
+                    break;
+                case MeasurementRepetition.RepeatIndefinitely:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, 0xff);
+                    break;
+                case MeasurementRepetition.Off:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, 0x00);
+                    break;
+            }
+
+            if (delay != -1)
+            {
+                WriteRegister(Register.MEASURE_DELAY, (byte)delay);
+
+                // Set mode to use custom delay.
+                var currentAcqSettings = AcquistionSettings;
+                currentAcqSettings |= AcquistionSettings.UseDefaultDelay;
+                AcquistionSettings = currentAcqSettings;
+            }
+            else
+            {
+                // Set mode to use default delay.
+                var currentAcqSettings = AcquistionSettings;
+                currentAcqSettings &= ~AcquistionSettings.UseDefaultDelay;
+                AcquistionSettings = currentAcqSettings;
+            }
+
+            // Kick it off with a single acquire command.
+            WriteRegister(Register.ACQ_COMMAND, 0x04);
+        }
+
+        /// <summary>
+        /// Set a new I2C address.
+        /// </summary>
+        /// <remarks>
+        /// Note, if the device is powered off or reset, the IC2 address will reset to the default address.
+        /// </remarks>
+        /// <param name="address">new address, valid values are 7-bit values with 0 in the LSB.</param>
+        public void SetI2CAddress(byte address)
```

Comment:
If this device is unusable after changing the address (because communication will be disrupted) then it should also Dispose (SetI2cAddressAndDispose)

Reply:
Makes sense.  Will add that.

Reply:
Just pushed up a fix for this, let me know if it makes sense.